src/bin/collapse-ghcprof.rs

```diff
@@ -0,0 +1,86 @@
+use std::io;
+use std::path::PathBuf;
+
+use clap::{ArgAction, ArgGroup, Parser};
+use env_logger::Env;
+use inferno::collapse::ghcprof::{Folder, Options, Source};
+use inferno::collapse::Collapse;
+
+#[derive(Debug, Parser)]
+#[clap(
+    name = "inferno-collapse-ghcprof",
+    about,
+    after_help = "\
+[1] This processes the .prof output of GHC (Glorious Haskell Compiler)
```

Comment:
```suggestion
[1] This processes the .prof output of GHC (Glasgow Haskell Compiler)
```
The G stands for Glasgow, not Glorious.

Reply:
Oh they seem to have dropped the glorious when I wasn't looking. Thanks 👍

Reply:
fixed

Reply:
It was never called "Glorious Haskell Compiler". The reason you were confused is probably because `ghc --version` says `The Glorious Glasgow Haskell Compilation System`.